dev/drivers/scripts/prep/cam/jevs_cam_hrrr_severe_prep.sh

```diff
@@ -4,7 +4,7 @@
 #PBS -q dev
 #PBS -A VERF-DEV
 #PBS -l walltime=0:30:00
```

Comment:
Job ran for ~1 min in my testing and I see in the parallel the max runtime it has is also ~1 min. Can we make the walltime 10 min to follow NCO's guideline for jobs run 0-2 min to have a walltime of 10 min?

Reply:
Sure, let me change that now

Reply:
Done 👍